paymentsheet/src/main/java/com/stripe/android/paymentsheet/injection/PaymentOptionsViewModelModule.kt

```diff
@@ -11,7 +11,9 @@ import dagger.Provides
 import javax.inject.Singleton
 import kotlin.coroutines.CoroutineContext
 
-@Module
+@Module(
+    subcomponents = [PaymentOptionsViewModelSubcomponent::class]
+)
```

Comment:
It might be helpful to make it clear that this is for the PaymentOptionsViewModelSubcomponent, not the PaymentOptionsViewModelComponent?